test/test_other.py

```diff
@@ -6411,7 +6411,7 @@ def test_dlopen_contructors(self):
 
       int main() {
         void (*check) (void);
-        void* h = dlopen("libside.wasm", RTLD_NOW|RTLD_GLOBAL);
+        void* h = dlopen("libside.wasm", RTLD_NOW);
```

Comment:
Is this because the test can run either way? Why change it in this PR, though? (I assume others could be similarly changed?)

Reply:
Just because its no longer needed.   I didn't find any others that could be removed